posthog/models/feature_flag/flag_matching.py

```diff
@@ -68,6 +69,17 @@
 ENTITY_EXISTS_PREFIX = "flag_entity_exists_"
 PERSON_KEY = "person"
 
+# Define which database to use for persons only.
+# This is temporary while we migrate persons to its own database.
+# It'll use `replica` until we set the PERSONS_DB_WRITER_URL env var
+DATABASE_FOR_PERSONS = (
+    "persons_db_writer"
+    if "persons_db_writer" in connections
+    else "replica"
+    if "replica" in connections and "decide" in settings.READ_REPLICA_OPT_IN
+    else "default"
+)  # Fallback if persons DB not configured
```

Comment:
this might be a premature optimization, but given that this connection string is only used in read requests, it might not hurt to use a replica persons DB URL instead of the writer, given that persons DB will still see significant write load, and there's no real reason not to just use a reader here.  Not really blocking, but could be a way to potentially distribute load if we still see contention for DB resources with this new database.